pkg/api/server/register_images.go

```diff
@@ -631,13 +631,18 @@ func (s *APIServer) registerImagesHandlers(r *mux.Router) error {
 	//    required: true
 	//    description: Name of image to push.
 	//  - in: query
-	//    name: tag
+	//    name: destination
 	//    type: string
-	//    description: The tag to associate with the image on the registry.
+	//    description: Allows for pushing the image to another destination than the image refers to.
```

Comment:
Suggestion: "...to a different destination than the image..."

Reply:
Done :)